ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-health-template.html

```diff
@@ -56,7 +56,9 @@
                                     <span class="column-title column-title-bottom">Alerts</span>
                                 </div>
                                 <div flex="30" layout="column" hide show-gt-md class="item-column md-list-item-text">
-                                    <span class="item-title">{{service.checkDate |  date : "MM/dd/yyyy 'at' h:mm"}}</span>
+                                    <span class="item-title">
+                                    {{'service-component-detail.time-locale' | translate:{date:service.checkDate} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:service.checkDate} }}
```

Comment:
Is it possible to merge this into a single translation? I don't understand the reason for splitting it into date, 'at', and time.